scss/ckeditor/skins/tao/scss/inc/_ck-icons.scss

```diff
@@ -1,4 +1,5 @@
 /* Do not edit */@import "inc/bootstrap";
+.cke .cke_top .cke_toolbar .cke_button_icon {background-image: none!important;}
```

Comment:
can it be done without !important?

Reply:
Unfortunately not it's overrides styles with `!important` property provided by plugin